test/Identity.IntegrationTest/Endpoints/IdentityServerTests.cs

```diff
@@ -179,6 +178,255 @@ public async Task TokenEndpoint_GrantTypePassword_WrongAuthEmailHeader_Fails()
         AssertHelper.AssertJsonProperty(root, "error_description", JsonValueKind.String);
     }
 
+    [Theory]
+    [BitAutoData(OrganizationUserType.Owner)]
+    [BitAutoData(OrganizationUserType.Admin)]
+    public async Task TokenEndpoint_GrantTypePassword_WithOwnerOrAdmin_WithSsoPolicyEnabled_WithEnforceSsoPolicyForAllUsersTrue_Throw(OrganizationUserType organizationUserType, int generatedUsername)
+    {
+        var organizationId = new Guid("5e77c2cd-f84b-4a27-affa-12c08d1adbda");
+        var deviceId = "92b9d953-b9b6-4eaf-9d3e-11d57144dfeb";
+        var username = $"{generatedUsername}@example.com";
+
+        var server = _factory.WithWebHostBuilder(builder =>
+        {
+            builder.UseSetting("globalSettings:sso:enforceSsoPolicyForAllUsers", "true");
+        }).Server;
+
+        await server.PostAsync("/accounts/register", JsonContent.Create(new RegisterRequestModel
+        {
+            Email = username,
+            MasterPasswordHash = "master_password_hash"
+        }));
+
+        await CreateOrganizationWithSsoPolicy(organizationId, username, organizationUserType, ssoPolicyEnabled: true);
+
+        var context = await server.PostAsync("/connect/token", new FormUrlEncodedContent(new Dictionary<string, string>
+        {
+            { "scope", "api offline_access" },
+            { "client_id", "web" },
+            { "deviceType", DeviceTypeAsString(DeviceType.FirefoxBrowser) },
+            { "deviceIdentifier", deviceId },
+            { "deviceName", "firefox" },
+            { "grant_type", "password" },
+            { "username", username },
+            { "password", "master_password_hash" },
```

Comment:
These look identical for each test (including the existing ones), can we extract the `Dictionary` or `FormUrlEncodedContent` object to a static class property for re-use?

Reply:
Or wrap the whole `server.PostAsync` call in a helper method is probably the cleanest way to do it